sources/dylan/tests/regressions.dylan

```diff
@@ -240,6 +240,15 @@ define test issue-440 ()
               map(identity, cis), cis);
 end;
 
+define test issue-1091 ()
```

Comment:
I realize this hasn't been the tradition in this file to date, but could you add this?
``// Issue 1091: remove on <range> can return an invalid object``
and/or a URL for the issue?

Reply:
Done.